java-checks/src/main/java/org/sonar/java/checks/SimpleClassNameCheck.java

```diff
@@ -56,16 +56,10 @@ public void visitNode(Tree tree) {
     CompilationUnitTree cut = (CompilationUnitTree) tree;
     cut.types().stream().filter(NOT_EMPTY_STATEMENT).map(t -> ((ClassTree) t).symbol()).forEach(this::checkSymbol);
     List<ImportTree> imports = cut.imports().stream().filter(NOT_EMPTY_STATEMENT).map(t -> (ImportTree) t).collect(Collectors.toList());
```

Comment:
It's unnecessary to collect here , and stream again few lines below. The same goes for `checkImports` argument. Just pass the `Stream` around